### DISCUSSION TITLE
ASIN not received for amazon.fr marketplace

### DISCUSSION THREAD

**Discussion by Nilanth** [amzn/selling-partner-api-models#3292] [General]
We are not getting **ASIN** in **GET_MERCHANT_LISTINGS_ALL_DATA** reports for the **amazon.fr** marketplace. Is this a default behaviour or a bug? For other marketplaces, ASIN key is received but only the **amazon.fr** we are not receiving it.

**Comment by gopkg-dev** [comment]
Same problem, france won't return asin https://github.com/amzn/selling-partner-api-docs/issues/577

**Reply by stefnats** [reply]
Hey there,

I answered in that issue. In short: I know this behavior for some time and you need to gather data with other reports or other Api calls.

Hope that helps!
